vue/components/ui/molecules/form-input/form-input.vue

```diff
@@ -148,13 +149,29 @@ export const FormInput = {
         footerMinWidth: {
             type: Number,
             default: null
+        },
+        tooltipProps: {
+            type: Object,
+            default: () => ({})
         }
     },
     computed: {
         headerStyle() {
             const base = { "min-width": `${this.headerMinWidth}px` };
             return base;
         },
+        hasTooltip() {
+            return Object.keys(this.tooltipProps).length !== 0;
+        },
+        headerProps() {

```

Comment:
We typically don't use this pattern of assigning multiple props at the same type let's keep it explicitly as before.